mobile/lib/features/trade/tradingview_candlestick.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/cupertino.dart';
+import 'package:webview_flutter/webview_flutter.dart';
+
+class TradingViewCandlestick extends StatefulWidget {
+  const TradingViewCandlestick({super.key});
+
+  static const String html = '''
+    <style>
+      #container {
+        width: 33%;
+        height: 33%;
+        transform: scale(3.0) translateX(33%) translateY(33%); 
+      }
+      
+      body {
+        overflow: hidden;
+      }
+    </style>
+    <div id="container">
+    <!-- TradingView Widget BEGIN -->
+    <div class="tradingview-widget-container" style="height:100%;width:100%">
+      <div class="tradingview-widget-container__widget" style="height:calc(100% - 32px);width:100%"></div>
+      <div class="tradingview-widget-copyright"><a href="https://www.tradingview.com/" rel="noopener nofollow" target="_blank"><span class="blue-text">Track all markets on TradingView</span></a></div>
```

Comment:
🔧 I guess we can remove that link.

Reply:
We could, but I'm not sure if we would be violating some usage policy by doing that